src/serviceWorker.js

```diff
@@ -0,0 +1,142 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
```

Comment:
I don't see you calling register or unregister anywhere, so do you even need this file?

Reply:
I deleted extra files and started getting console errors so tried adding this back; didn't fix them - it looks like I'd need to eject the app in order to adjust Webpack configuration to fix them.